### PR TITLE
Clean up pages and test features

### PR DESCRIPTION
**Pull request by koredefashokun**
This PR fixes bugs in the UI, and makes sure that Snapshot information is displayed correctly.

### CODE REVIEW

**Comment by koredefashokun** [comment]
@clementbalestrat I created this draft PR, so you could view all the changes in one place. I'm not sure if this project has deploy previews set up.

File: pages/grants/[id].tsx [hunk 1]
```diff
@@ -7,78 +7,63 @@ import { Footer } from "../../components/Footer";
...
 import { useState } from "react";
 import useGetSnapshotSpace from "../../queries/useGetSnapshotSpace";
-import useGetSnapshotProposals from "../../queries/useGetSnapshotProposals";
+import useGetSnapshotProposals, {
+  Proposal,
+} from "../../queries/useGetSnapshotProposals";
 import useGetSnapshotVotes from "../../queries/useGetSnapshotVotes";
 import Moment from "react-moment";
-import { isConnected, onboard, state } from "../../services/Wallet";
+// import { isConnected } from "../../services/Wallet";
```
Comment (0xclem) [inline pages/grants/[id].tsx:15]: Can you remove all the things you commented out?

File: pages/grants/[id].tsx [hunk 2]
```diff
@@ -7,78 +7,63 @@ import { Footer } from "../../components/Footer";
...
 
-  console.log(proposal);
-
-  let userAddress = "";
+  let userAddress = walletAddress;
 
-  if (isConnected()) {
-    console.log("onboard.state()");
-    console.log(onboard.getState());
+  // if (isConnected()) {
```
Comment (0xclem) [inline pages/grants/[id].tsx:43]: same here. If you think we'll never need it then remove it please

File: pages/grants/[id].tsx [hunk 3]
```diff
@@ -88,25 +73,41 @@ export default function Grant() {
       let vote = "to vote";
 
       console.log("userAddress");
```
Comment (0xclem) [inline pages/grants/[id].tsx:75]: let's remove all the console.log in the code

File: pages/grants/[id].tsx [hunk 4]
```diff
@@ -88,25 +73,41 @@ export default function Grant() {
...
-        vote = choices[votes[adminAddress].choice - 1];
-
-        console.log(vote);
+      console.log(onboard?.getState().address);
+
+      if (votes && votes[proposal.id]) {
+        const voteIndex = votes[proposal.id][adminAddress]?.choice - 1;
+        if (choices[voteIndex]) {
+          vote = choices[voteIndex];
+          // (voteIndex === 0 ? yesVotes : noVotes)++;
```
Comment (0xclem) [inline pages/grants/[id].tsx:82]: Remove if not needed

**Review by 0xclem — APPROVED** [review]
LGTM. A few minor comments

**Pull request merged**